Tests/NIOTests/ByteBufferTest.swift

```diff
@@ -1939,6 +1939,34 @@ class ByteBufferTest: XCTestCase {
         XCTAssertEqual([], self.buf.readBytes(length: self.buf.readableBytes))
     }
 
+    func testBufferViewFirstIndex() {
+          self.buf.clear();
```

Comment:
nit: extra semicolon

Reply:
Yes, 6 of them.  I will remove.  I also notice my indent doesn't match the file either which I will also resolve.